code/modules/client/client procs.dm

```diff
@@ -143,6 +140,22 @@ var/next_external_rsc = 0
 
 	. = ..()	//calls mob.Login()
 
+	if (byond_version < config.client_error_version)		//Out of date client.
+		src << "<span class='danger'><b>Your version of byond is too old:</b></span>"
+		src << config.client_error_message
+		src << "Your version: [byond_version]"
+		src << "Required version: [config.client_error_version] or later"
+		if (holder)
```

Comment:
Because while testing, I set it to 530 by accident then I couldn't get in to fix it and I thought the idea that a rogue admin could do the same to prevent any admin from coming in to stop them was a bad idea.